yo1k/sandbox/postgresql/pgexercises/psycopg_basic.py

```diff
@@ -0,0 +1,78 @@
+from collections.abc import Sequence, MutableSequence
+from typing import Any
+import psycopg2  # type: ignore
+from psycopg2 import sql
+
+
+def get_full_table(*table_name_parts: str) -> MutableSequence[Sequence[Any]]:
+    """Fetch a whole table from `exercises` database and return tuple of all records."""
```

Comment:
_"return tuple of all records"_ - something is wrong here.

Reply:
Fetch -> Fetches
return -> returns

Reply:
Done.